pool/paymentmgr.go

```diff
@@ -726,13 +710,18 @@ func (pm *PaymentMgr) payDividends(ctx context.Context, height uint32, treasuryA
 		return nil
 	}
 
-	if pm.isProcessing() {
+	pm.mtx.Lock()
+	if pm.processing {
+		pm.mtx.Unlock()
```

Comment:
Looks like the `return` got lost?

Reply:
Yep. Lost it in a rebase it seems.  Added back.